City-of-Helsinki/berth-reservations-admin
# VEN-718 | Implement WinterStorageAreaView

## Description :sparkles:

Initial implementation for the single winter storage area view.

### Closes :no_good_woman:

[VEN-718](https://helsinkisolutionoffice.atlassian.net/browse/VEN-718)

## Testing :alembic:

### Automated tests :gear:️

- `src/common/winterStorageAreaCard/__tests__/WinterStorageAreaCard.test.tsx`
- `src/features/winterStorageAreaView/__tests__/WinterStorageAreaViewContainer.test.tsx`

### Manual testing :construction_worker_man:

1. Open the winter storage areas list page
2. Click on one of the items on the list
3. This should open a new page displaying details for the winter storage area
